### PR TITLE
Add Shadows over Innistrad with release date

### PR DESCRIPTION
**Pull request by Kerrick**
what_s_in_standard_

### CODE REVIEW

**Comment by Nightfirecat** [comment]
LGTM :shipit:

File: index.html [hunk 1]
```diff
@@ -55,6 +55,17 @@
...
           </span>
         </ul>
+        <ul class="list-group">
+          <li class="list-group-item gray">
+            Shadows over Innistrad
+          </li>
+          <li class="list-group-item gray">
+            ...
+          </li>
+          <span class="label  label-default">
```
Comment (glacials) [inline index.html]: oops, you have a double space here!

**Comment by glacials** [comment]
Thanks! Can you also update the text under the *Khans of Tarkir* block to say "Until April 8, 2016" now that we have an exact drop date?

**Comment by Kerrick** [comment]
Fixed and rebased!

**Comment by glacials** [comment]
Perfect! :ok_hand:

**Pull request merged**